### PR TITLE
Tests for stridedarray/sparse mult

### PR DESCRIPTION
**Pull request by kshyatt**
Trying to get at [this](https://codecov.io/gh/kshyatt/julia/src/master/stdlib/SparseArrays/src/linalg.jl#L118) and the method above it.

### CODE REVIEW

**Comment by ViralBShah** [comment]
@kshyatt Nice to have your eyes on the sparse libraries!

**Review by Sacha0 — APPROVED** [review]
Thanks @kshyatt! :)

File: stdlib/SparseArrays/test/sparse.jl [hunk 1]
```diff
@@ -340,6 +340,13 @@ end
         f = Diagonal(rand(n))
         @test Array(a*f) == Array(a)*f
         @test Array(f*b) == f*Array(b)
+        A = rand(2n, 2n)
+        sA = view(A, 1:2:2n, 1:2:2n)
+        @test Array(sA*b) == Array(sA)*Array(b)
```
Comment (andreasnoack) [inline stdlib/SparseArrays/test/sparse.jl]: This has to be tested with `≈`.

**Review by andreasnoack — CHANGES_REQUESTED** [review]

**Comment by kshyatt** [comment]
@andreasnoack is that better? Should all of them be approx?

**Review by andreasnoack — APPROVED** [review]

**Pull request merged**